src/legacy/server/kbn_server.js

```diff
@@ -175,7 +175,10 @@ export default class KbnServer {
 
   applyLoggingConfiguration(settings) {
     const config = Config.withDefaultSchema(settings);
-    const loggingOptions = loggingConfiguration(config);
+    const loggingOptions = getLoggingConfiguration(
+      config.get('logging'),
+      config.get('ops.interval')
+    );
```

Comment:
It's a bit odd that we register `@elastic/good` in kbn-legacy-logging/src/setup_logging.ts but reconfigure it here. I'd suggest moving this to the package as well.